docs/home/howto/LOGGING.md

```diff
@@ -210,7 +210,7 @@ PUT _template/template_01
 After applying this policy, every new index created under this one will apply to it. There is also possibility to apply
 policy to already existing policies by assigning them to policy in Index Management Kibana panel.
 
-## How to export Elasticsearch data to csv format
+## Export Elasticsearch data to CSV format
```

Comment:
If you want to change this header you have change reference here: https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/HOWTO.md